lxml-stubs/etree.pyi

```diff
@@ -44,21 +50,27 @@ _XPathObject = Union[
     bool,
     float,
     _AnySmartStr,
-    _AnyStr,
+    _StrResult,
     List[
         Union[
             "_Element",
             _AnySmartStr,
-            _AnyStr,
-            Tuple[Optional[_AnyStr], Optional[_AnyStr]],
+            _StrResult,
+            Tuple[Optional[_StrResult], Optional[_StrResult]],
         ]
     ],
 ]
 _AnyParser = Union["XMLParser", "HTMLParser"]
-_ListAnyStr = Union[List[str], List[bytes]]
-_DictAnyStr = Union[Dict[str, str], Dict[bytes, bytes]]
-_Dict_Tuple2AnyStr_Any = Union[Dict[Tuple[str, str], Any], Tuple[bytes, bytes], Any]
-_xpath = Union["XPath", _AnyStr]
+if sys.version_info[0] >= 3:
+    _ListAnyStr = List[str]
+    _DictAnyStr = Dict[str, str]
+else:
+    _ListAnyStr = Union[List[str], List[bytes]]
+    _DictAnyStr = Union[Dict[str, str], Dict[bytes, bytes]]
+_StrOrBytes = Union[str, bytes]
+_ValueType = Union[str, bytes, QName]
+_InputDictAnyStr = Dict[_StrOrBytes, _StrOrBytes]
```

Comment:
Add type aliases for parameters that allow str or bytes on both Python 2 and Python 3.